test/suites/standard/kustomize.robot

```diff
@@ -241,6 +241,7 @@ Restore Default Config
     ${is_ostree}=    Is System OSTree
     IF    ${is_ostree}
         Reboot MicroShift Host
+        Wait For MicroShift Service
```

Comment:
Maybe we should do this inside "Wait For MicroShift" so we do it everywhere we restart MicroShift?

Reply:
That make sense to me, updated.